src/network/networkctl.c

```diff
@@ -106,6 +106,7 @@ static void setup_state_to_color(const char *state, const char **on, const char
 
 typedef struct LinkInfo {
         char name[IFNAMSIZ+1];
+        char netdev_kind[IFNAMSIZ+1];
```

Comment:
Could you tell me where the maximum length of `kind` is defined in kernel?

Reply:
it's defined in one place like ```MODULE_NAME_LEN 64 - sizeof(Elf_Addr))``` so I think 54 should be OK.